.gitignore

```diff
@@ -1,5 +1,7 @@
-/bower_components/
-/node_modules/
-/output/
-/.psci*
-/src/.webpack.js
+/.*
+!/.gitignore
+!/.travis.yml
+bower_components/
+node_modules/
+output/
+
```

Comment:
Also removing the slash-prefix for `bower_components` and `output` ensure git won't capture that stuff from the example subproject(s) too.